Module not found: Error: [CaseSensitivePathsPlugin], jQuery

#### steps:
1. I faced this problem.
Module not found: Error: [CaseSensitivePathsPlugin] `C:\Users\[UserName]\[Something_Dir]\[MyProject]\node_modules\jQuery\dist\jquery.js` does not match the corresponding path on disk `jquery`.

2. I think its trouble mean -> 'jquery' required 'jQuery'
```
/dist/summernote-lite.js (line:15)
```

follow the code.
```js
module.exports = factory(require("jQuery"));
```

I think its right.
```js
module.exports = factory(require("jquery"));
```
'Q' -> 'q'

can you check it, please? Thank!

#### Settings
1. Operating System: Windows 10 and Ubuntu 16.04 LTS

2. Browser and Version: Chrome

3. Summernote Version: v0.8.14 (lite)

4. Package manager: yarn

and.. I use these dependencies.
 - Vue CLI 3
 - jquery^3.4.1



5. Please feel free to email me. (essess1541@gmail.com).

#### screenshot of issue
![image](https://user-images.githubusercontent.com/24954436/71611970-68414c80-2be0-11ea-868b-7925b3a7e93b.png)

getting same issue since yesterday

```
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Use equal casing. Compare these module identifiers:
* node_modules\jQuery\dist\jquery.js
    Used by 1 module(s), i. e.
    node_modules\summernote\dist\summernote-lite.js
Module not found: Error: Can't resolve 'jQuery' in '/node_modules/summernote/dist'

```

--- a/config/webpack.production.js
+++ b/config/webpack.production.js
@@ -65,7 +65,7 @@ module.exports = {
 
   externals: {
     jquery: {
-      root: 'jquery',
+      root: 'jQuery',
       commonjs: 'jquery',
       commonjs2: 'jquery',
       amd: 'jquery',

